ksql-rest-app/src/test/java/io/confluent/ksql/rest/integration/RestApiTest.java

```diff
@@ -179,7 +179,7 @@ public void shouldDeleteTopic() {
       // When:
       RestIntegrationTestUtil.makeKsqlRequest(
           REST_APP,
-          "TERMINATE QUERY CSAS_X_0; DROP STREAM X DELETE TOPIC;");
+          "TERMINATE QUERY CSAS_X_1; DROP STREAM X DELETE TOPIC;");
```

Comment:
Has this Id changed because its now using the offset?

Reply:
Yup, in the setUpClass(), a CREATE STREAM is called so when the query is made in this test, it would be at offset one